app/assets/javascripts/react_components/book.js.jsx

```diff
@@ -23,13 +23,13 @@ var CartButton = React.createClass({
     if (_.findWhere(this.props.cart, {id: this.props.book.id})) {
       return (
         <button className="btn cart-button" onClick={this.handleClick}>
-        Remove from Cart
+          Remove from Booklist
         </button>
       );
     } else {
       return (
         <button className="btn cart-button" onClick={this.handleClick}>
-        Add to Cart
+          Add to Booklists
```

Comment:
`Add to Booklist`

Just because the other one is singular. Ideally this would depend on the number of booklists selected.